### ISSUE TITLE
Custom nametag malfunction

### ISSUE THREAD

**Issue by otocin** [codehz/ElementZero#64]
custom Nametag is not working properly.

It is not cleared or renamed to another name after setting up the name once.

**Comment by ub0nFPS** [comment]
Describe the issue in detail...
How did you get this error to occur?

**Comment by otocin** [comment]
After setting the name with the command "/custom-name set nametag User "xxxxxx"", I tried to change it to a different name, but it did not change.

I also tried "/custom-name clear User", but it remained the same.

**Comment by otocin** [comment]
I've just tried again on this problem.

When I entered the command "/custom-name clear@a" and reconnected to the server, I confirmed that the custom nameag of all users was gone.

However, it did not work normally when a command was entered as a target for a specific user or me (@s).

**Comment by lightyisreal** [comment]
@otocin For me it just sets the nickname above, but not in chat :(

**Comment by otocin** [comment]
@ItzLightyHD Yes. I don't think 'prefix / postfix' and 'nametag' apply at the same time.
